### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by MoAlali**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/Varies/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "server_name": "Varies",
+  "nice_name": "Varies",
+  "direct_ip": "Play.Varies.Ae",
+  "server_wildcards":[
+     "%.Varies.ae"
+  ],
+  "social": {
+    "web": "https://Varies.ae",
```
Comment (DoJapHD) [inline minecraft_servers/Varies/manifest.json:9]: Can't access website; if non-existent, remove

File: minecraft_servers/Varies/manifest.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "web_shop": "https://Varies.ae/store",
```
Comment (DoJapHD) [inline minecraft_servers/Varies/manifest.json:10]: Can't access website; if non-existent, remove

File: minecraft_servers/Varies/manifest.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "discord": "https://Varies.ae/discord"
```
Comment (DoJapHD) [inline minecraft_servers/Varies/manifest.json:11]: Can't access website; Discord link cannot be redirect; use https://discord.gg/[YOUR INVITE]

File: minecraft_servers/Varies/manifest.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "server_name": "Varies",
+  "nice_name": "Varies",
+  "direct_ip": "Play.Varies.Ae",
```
Comment (DoJapHD) [inline minecraft_servers/Varies/manifest.json:4]: Change to lowercase

File: minecraft_servers/Varies/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/Varies/background@2x.png:1]: Compress image

File: minecraft_servers/Varies/background.png
Comment (DoJapHD) [inline minecraft_servers/Varies/background.png:1]: Compress image

**Review by DoJapHD — CHANGES_REQUESTED** [review]
Fix images (see failed workflow), rename folder to lowercase

File: minecraft_servers/Varies/manifest.json (hunk 1 above)
Comment (MoAlali) [reply minecraft_servers/Varies/manifest.json:9]: It is there but in maintenance mode right now

File: minecraft_servers/Varies/manifest.json (hunk 2 above)
Comment (MoAlali) [reply minecraft_servers/Varies/manifest.json:10]: It is there but in maintenance mode right now

File: minecraft_servers/Varies/manifest.json (hunk 3 above)
Comment (MoAlali) [reply minecraft_servers/Varies/manifest.json:11]: It is there but in maintenance mode right now

**Pull request closed without merging**